src/app/products/manage-tax-components/edit-tax-component/edit-tax-component.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="taxComponentForm" (ngSubmit)="submit()">
+
```

Comment:
![Screenshot from 2020-05-11 12-22-10](https://user-images.githubusercontent.com/54709463/81532494-17a5e100-9382-11ea-8477-530b4932eea0.png)
@spider0061 there are 2 more non editable fields, please add.

Reply:
@karantakalkar As per one of my PR and discussion with Abhay, he suggested me to keep the non-editable fields as form input itself but it will be deactivated. We need to confirm is that the consistency maintained in the application. Then its good to go.

@abhaychawla Can you please guide us through the design?